app/views/simplified-account/settings/email-notifications/index.njk

```diff
@@ -71,9 +71,11 @@
             text: "Change",
             classes: "govuk-link--no-visited-state",
             visuallyHiddenText: "Refund emails settings"
-          } if (isServiceAdmin and emailCollectionMode !== 'OFF') else {}
+          } if (isAdminUser and emailCollectionMode !== 'OFF') else {}
         ]
       }
       }
     ]}) }}
+
+  <p class="govuk-body"><a class="govuk-link" href="{{ templatesHref }}" id="templates-link">View email templates and add a custom paragraph</a></p>
```

Comment:
one of the designs shows this saying just 'View email templates' but not sure if this is a mistake in the design